content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+## Overview
+Datadog Synthetics monitors your applications and API endpoints via simulated user requests and browser rendering. Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing, as experienced by your users.
+
+{{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" responsive="true">}}
+
+This guide shows you how to set up your first Synthetics test with Datadog. Follow the sections below to configure your API, browser test, or private location for internal-facing apps or private URLs.
+
+## Prerequisites
+
+The browser tests can be only recorded from **[Google Chrome][1]**. Datadog will prompt you to download this extension during the creation of the test. You can also download the [Datadog Record Test extension for Google Chrome][2].
+
+## Create a Datadog account
+
+If you haven't already, create a [Datadog account][3].
+
+## Configure your first test
+
+Synthetics offers three different ways to monitor your applications and API endpoints: API test, browser test, and private locations.
```

Comment:
Synthetics offers two different ways to monitor your applications: API tests to monitor the uptime of your API endpoints, and browser tests to check key user journeys. Your tests can be run either from managed locations or from private locations.